src/sbt-test/sbt-reproducible-builds/assembly/build.sbt

```diff
@@ -0,0 +1,4 @@
+scalaVersion := "2.12.7"
+
+enablePlugins(ReproducibleBuildsPlugin)
+enablePlugins(AssemblyPlugin)
```

Comment:
Apparently I have to enable the Assembly for the reproducible hook to work, although the AssemblyPlugin is an auto plugin?

Reply:
That is indeed pretty weird.

Without this explicit call the assembly plugin was enabled, but indeed `assembly` called the original task instead of the 'reproducible' one.

Putting a `$ pause` in the scripted test right before the failing check, started an interactive sbt shell and did `inspect assembly`. Sure enough this confirmed `assembly` was the original `assembly` from `AssemblyPlugin.scala:57`.

I wonder if it's some sort of ordering or scoping thing?

Reply:
I can confirm that changing the order here matters, it won't work if the order was:
```
enablePlugins(AssemblyPlugin)
enablePlugins(ReproducibleBuildsPlugin)
```